### DISCUSSION TITLE
How to balance workspaces between hosts?

### DISCUSSION THREAD

**Discussion by fuale** [coder/coder#10245] [General] [answered]
I have two VMs, each with a running Docker daemon. I would like to automatically allocate workspaces upon their creation to the host with the fewest workspaces. Is this currently possible or does it need to be implemented? Alternatively, if user can choose in which host workspace should be created, maybe show them load of a server or something similar?

**Comment by matifali** [comment, marked as answer]
You may switch to creating a kubernetes cluster out of these two VMs and run your workspaces as pods. Kubernetes has built-in load management.
